### PR TITLE
PixArt-Sigma basic training/inference script

### PR DESCRIPTION
**Pull request by zhtmike**
# What does this PR do?

Fixes # (issue)

Adds # (feature)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

File: examples/pixart_sigma/README.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+    --image_dir path_to_your_image_directory \
+    --use_parallel True
+```
+- Remember to
+    - Replace `path_to_your_label_file` with the actual path to your label JSON file.
+    - Replace `path_to_your_image_directory` with the directory containing your images.
+
+## Benchmark
+
+### Training
```
Comment (SamitHuang) [inline examples/pixart_sigma/README.md:152]: pls add qualititative or visual evaluation results

**Review by CaitinZhao — APPROVED** [review]

File: mindone/trainers/came.py [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+                Parameter(
+                    ops.zeros_like(x, dtype=ms.float32),
+                    name=x.name + "_m",
+                    requires_grad=False,
+                )
+                for x in self._parameters
+            ]
+        )
+
+    @ms.jit
```
Comment (SamitHuang) [inline mindone/trainers/came.py:232]: why add jit?

File: examples/pixart_sigma/pixart/pipelines/infer_pipeline.py [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+        """
+        y = self.vae.decode(x / self.scale_factor)[0]
+        y = ops.clip_by_value((y + 1.0) / 2.0, clip_value_min=0.0, clip_value_max=1.0)
+
+        # (b 3 H W) -> (b H W 3)
+        y = ops.transpose(y, (0, 2, 3, 1))
+
+        return y
+
+    @ms.jit
```
Comment (SamitHuang) [inline examples/pixart_sigma/pixart/pipelines/infer_pipeline.py:91]: why add jit?

File: mindone/trainers/came.py (hunk 2 above)
Comment (zhtmike) [reply mindone/trainers/came.py:232]: almost all mindspore optimizers implementation using `jit` to accelerate training. Here it is following the same practice.

File: examples/pixart_sigma/pixart/pipelines/infer_pipeline.py (hunk 3 above)
Comment (zhtmike) [reply examples/pixart_sigma/pixart/pipelines/infer_pipeline.py:91]: it is faster in batch inference in pynative mode, no effect in graph mode

File: examples/pixart_sigma/README.md (hunk 1 above)
Comment (zhtmike) [reply examples/pixart_sigma/README.md:168]: done

**Review by SamitHuang — APPROVED** [review]

**Review by zhanghuiyao — APPROVED** [review]

**Review by geniuspatrick — APPROVED** [review]

**Review by vigo999 — APPROVED** [review]
